demos/applications/apa102.cpp

```diff
@@ -75,9 +75,7 @@ void application(resource_list& p_map)
   hal::display::apa102_pixel white{ .blue = 0xFF, .green = 0xFF, .red = 0xFF };
   std::array<hal::display::apa102_pixel, led_count> all_off{};
 
-  std::array<hal::display::apa102_pixel, led_count> rgb_array = {
-    { white, blue, green, red }
-  };
+  // std::array<hal::display::apa102_pixel, led_count> rgb_array = { { blue } };
```

Comment:
Should we keep the demo this way or convert it back to what was originally?

Reply:
You reverted this back but it would be nice if this was made generic, so it could work with 7 or 10 LEDs and behave as you'd expect.

Reply:
Nvm, you removed the code.